packages/common/src/constants.ts

```diff
@@ -185,8 +185,7 @@ export enum PUBLIC_GRAPHQL_METHODS {
 }
 
 // seconds
-export const CONNECTION_RETRY_STEP = RELATE_IS_TESTING ? 12 : 4;
-export const MAX_CONNECTION_RETRIES = 5;
+export const MAX_NUMBER_OF_DBMS_TO_BE_ONLINE_ATTEMPTS = 60;
```

Comment:
Maybe omit the number part, given that it's implied by the type?
```suggestion
export const MAX_DBMS_TO_BE_ONLINE_ATTEMPTS = 60;
```